### ISSUE TITLE
Nested fields add or remove - not triggering change event

### ISSUE THREAD

**Issue by prajeeshbs** [foxhound87/mobx-ajv-form#538]
I have a  mobx react form with nested fields.
There is a status button at the bottom. When this button is clicked, `form.changed` is used to detect if a change is made to the form. This works pretty well for field changes and returns true when any field is modified.

But it fails under the following scenarios

1) if a new record is added using the add button and no other changes are made to the form and button is clicked - `form.changed` returns false.
2) if an existing record is removed using the remove button and no other changes are made to the form and button is clicked - `form.changed` returns false.

Any idea on how to fix this?

Edit1:
```
const fields = [
  'items[]',
  'items[].qty',
  'items[].amount'
];

const form = new Form({ fields });
const items = form.$('items');
items.add({ qty: 0, amount: 0  }); 
//form.changed = false
```

**Comment by curlybom** [comment]
@foxhound87 any update or solution for this issue? I've faced this problem too. My nested fields don't trigger changes and updates

**Comment by foxhound87** [comment]
Available in next pre-release 5.2.0
